modules/UI/videolayout/FilmStrip.js

```diff
@@ -3,6 +3,15 @@
 import UIEvents from "../../../service/UI/UIEvents";
 import UIUtil from "../util/UIUtil";
 
+/**
+ * Contains sizes of thumbnails
+ * @type {{SMALL: number, MEDIUM: number}}
+ */
+const ThumbnailSizes = {
+    SMALL: 60,
+    MEDIUM: 80
+};
```

Comment:
This doesn't look like a very scalable solution and looks a little bit like a hack. If we manage to use some responsive font size solution (see above) it will be better.